grub2-efi/install

```diff
@@ -64,6 +64,10 @@ if [ "$SYS__BOOTLOADER__SECURE_BOOT" = "yes" -a "$target" != "arm64" ] ; then
     exit 1
   fi
 elif [ -x /usr/sbin/grub2-install ] ; then
+  # Use '--suse-signed-grub' when shim is not used (aarch64 case)
+  if [ "$SYS__BOOTLOADER__SECURE_BOOT" = "yes" ]; then
+     append="$append --suse-signed-grub"
+  fi
```

Comment:
is following lines with cp still needed? I hope that grub2 changes makes it unnecessary.

Reply:
Of courses, cp lines are not needed anymore. Fixed in new push.